LoopKitTests/CarbMathTests.swift

```diff
@@ -171,7 +171,7 @@ class CarbMathTests: XCTestCase {
     func testDynamicGlucoseEffectAbsorptionNoneObserved() {
         let inputICE = loadICEInputFixture("ice_35_min_input")
         let carbEntries = loadCarbEntryFixture()
-        let output = loadCOBOutputFixture("dynamic_glucose_effect_none_observed_output")
+        let output = loadEffectOutputFixture("dynamic_glucose_effect_none_observed_output")
```

Comment:
This is a small naming clean up from a previous PR - since the 'fixture loader' function pulls units from the fixture, using `loadCOBOutputFixture` actually works fine (though it's confusing to look at)